docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -105,7 +132,8 @@ public class IMDBSentiment
 
             /// <summary>
             /// This is a variable length vector designated by VectorType attribute.
-            /// Variable length vectors are produced by applying operations such as 'TokenizeWords' on strings
+            /// Variable length vectors are produced by applying operations such as
+            /// 'TokenizeWords' on strings
```

Comment:
This line has some space for storing tokens in the line right below.

Reply:
Changed on new commit